examples/ordered_wordcount/run_ordered_wordcount.py

```diff
@@ -36,7 +36,7 @@ def main():
             "--textfile",
             f"{curr_dir}/wizard_of_oz_book.txt",
             "--name",
-            "oz"
+            "oz",
```

Comment:
I'm noticing these in a couple of places. Why the list-ending commas?

Reply:
Black does this. I don't like it but it's a default setting for the formatter.